README.md

```diff
@@ -45,11 +45,13 @@ auth_jwt_loginurl "https://yourdomain.com/loginpage";
 auth_jwt_enabled on;
 auth_jwt_algorithm HS256; # or RS256
 auth_jwt_validate_email on;  # or off
+auth_jwt_use_keyfile off; # or on
+auth_jwt_keyfile_path "/app/pub_key";
```

Comment:
Is this a valid default path? All my docker images are inside `/app` working directory, just a habit 😄

Reply:
I think there perhaps shouldn't be a default, and it would be an error case to have `auth_jwt_use_keyfile on` and `auth_jwt_keyfile_path` not supplied (e.g. force the user to supply a path).

Reply:
Makes sense, but there could also be a `auth_jwt_key` together with the `auth_jwt_keyfile_path` which adds ambiguity to what key is being used. Maybe an enum field `auth_jwt_key_location` (with values `inline` or `file`) would solve it also for future situations (for example having a remote keystore where the key is located).